example/provider-local/garden/local/kustomization.yaml

```diff
@@ -6,5 +6,5 @@ resources:
 - controllerregistration.yaml
 
 patches:
-- path: patch-imagevector-overwrite.yaml
 - path: patch-controllerdeployment-prow.yaml
+- path: patch-imagevector-overwrite.yaml
```

Comment:
Do we need the same change in https://github.com/gardener/gardener/blob/ee40d427febdf2d37e2766c0cbdf47612786f10c/example/provider-local/garden/operator/kustomization.yaml#L7-L9?

Reply:
yes, it is the same here 👍